src/txn/txn_rollback_to_stable.c

```diff
@@ -1676,7 +1672,7 @@ __rollback_to_stable_btree_apply(
 
     if (perform_rts || max_durable_ts > rollback_timestamp || prepared_updates ||
       !durable_ts_found || has_txn_updates_gt_than_ckpt_snap) {
-        ret = __wt_session_get_dhandle(session, uri, NULL, NULL, 0);
+        ret = __wt_session_get_dhandle(session, uri, NULL, NULL, WT_DHANDLE_DISCARD);
```

Comment:
Can we get a comment explaining what the significance of `WT_DHANDLE_DISCARD` is here?

Reply:
Fixed in 75c3444ba.